### PR TITLE
Change externalElasticsearchHost to externalElasticsearchUrl

### PR DESCRIPTION
**Pull request by Felixnoo**
Signed-off-by: Felixnoo <felixliu@kubesphere.io>

I think some relevant documents also need to be updated according to the changes made in #2285.

/cc @kubesphere/sig-observability @wanjunlei @wenchajun 
Please review this PR, thanks.

### CODE REVIEW

File: content/en/docs/pluggable-components/events.md [hunk 1]
```diff
@@ -47,7 +47,7 @@ By default, KubeKey will install Elasticsearch internally if Events is enabled.
       elasticsearchDataVolumeSize: 20Gi    # The volume size of Elasticsearch data nodes.
       logMaxAge: 7                     # Log retention day in built-in Elasticsearch. It is 7 days by default.
       elkPrefix: logstash              # The string making up index names. The index name will be formatted as ks-<elk_prefix>-log.
-      externalElasticsearchHost: # The Host of external Elasticsearch.
+      externalElasticsearchUrl: # The URL of external Elasticsearch.
```
Comment (wanjunlei) [inline content/en/docs/pluggable-components/events.md:50]: `The URL of external Elasticsearch` => `The Host of external Elasticsearch`

`Host` is more accurate, please also modify other places.

File: content/en/docs/pluggable-components/events.md [hunk 2]
```diff
@@ -132,7 +132,7 @@ By default, Elasticsearch will be installed internally if Events is enabled. For
...
```
Comment (wanjunlei) [inline content/en/docs/pluggable-components/events.md:135]: `The URL of external Elasticsearch` => `The Host of external Elasticsearch`

**Review by wanjunlei — CHANGES_REQUESTED** [review]

File: content/en/docs/pluggable-components/events.md (hunk 1 above)
Comment (Felixnoo) [reply content/en/docs/pluggable-components/events.md]: Thanks and updated in [this commit](https://github.com/kubesphere/website/pull/2289/commits/3869e9d2b36c6ca22851c11ee99c5a4dbf3a6e28).

**Review by wanjunlei — APPROVED** [review]

**Pull request merged**